config/jobs/kubernetes/cloud-provider-vsphere/cloud-provider-vsphere-config-1.26-minus.yaml

```diff
@@ -7,7 +7,7 @@ presubmits:
     run_if_changed: '\.go$|hack/check-format\.sh'
     decorate: true
     branches:
-    - ^release-1.2[2-6]$
```

Comment:
I think we just need to change the `pull-cloud-provider-vsphere-e2e-test-1-26-minus` job's branch to 1.26 and keep others untouched because those jobs previously didn't run on VMware-owned env.